code/game/objects/items/melee/baton.dm

```diff
@@ -437,7 +437,7 @@
 	var/throw_stun_chance = 35
 	var/obj/item/stock_parts/cell/cell
 	var/preload_cell_type //if not empty the baton starts with this type of cell
-	var/cell_hit_cost = 1000
+	var/cell_hit_cost = STANDARD_CELL_CHARGE * 0.1
```

Comment:
```suggestion
	var/cell_hit_cost = STANDARD_CELL_CHARGE
```

The standard cell (not to be confused with the high capacity one, which is 10MJ) only has 1MJ. The standard cell charge before the energy change was 1,000.

Reply:
Oof.